<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>887</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>871</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>100</y>
        <width>160</width>
        <height>24</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>停止位</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_StopBit">
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>140</y>
        <width>160</width>
        <height>24</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>数据位</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_DataBit">
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_ClearData">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>290</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>清空数据</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>180</y>
        <width>160</width>
        <height>24</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>校验位</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_CheckBit">
         <item>
          <property name="text">
           <string>No</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Even</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Odd</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Space</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mark</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_connect">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>220</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>连接串口</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>160</width>
        <height>24</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>波特率</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_BaudRate">
         <property name="currentText">
          <string>9600</string>
         </property>
         <item>
          <property name="text">
           <string>2400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4800</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>19200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>115200</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QCheckBox" name="checkBox_hex">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>260</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>16进制显示</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>340</y>
        <width>160</width>
        <height>131</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_PIDSetting">
         <property name="text">
          <string>PID修改</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_7">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>160</width>
        <height>24</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>串口号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_SerialPort"/>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>340</y>
        <width>591</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string>发送</string>
      </property>
      <widget class="QTextEdit" name="textEdit_dataSend">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>551</width>
         <height>101</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_send">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>140</y>
         <width>80</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
      <zorder>textEdit_dataSend</zorder>
      <zorder>textEdit_dataSend</zorder>
      <zorder>pushButton_send</zorder>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>10</y>
        <width>591</width>
        <height>321</height>
       </rect>
      </property>
      <property name="title">
       <string>接收</string>
      </property>
      <widget class="QTextEdit" name="textEdit_dataRec">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>551</width>
         <height>281</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>220</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>扫描串口</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>887</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
